iac-extensions/arm/src/test/java/org/sonar/iac/arm/tree/impl/bicep/MemberExpressionImplTest.java

```diff
@@ -137,6 +143,21 @@ void shouldParseMemberExpressionWithIdentifier() {
     assertThat(((TextTree) tree.children().get(1)).value()).isEqualTo(":");
   }
 
+  @Test
+  void shouldParseMemberExpressionDoubleColonWithIdentifier() {
+    String code = code("virtualNetwork::subnet1");
```

Comment:
Nitpic. Calling a method `code()` doesn't have sense here ;-) (and in other lines in this file too)

Reply:
Good catch :sweat_smile: I made a small refactoring for this class, it was also using a manually created parser, removed it.